src/Migrations/Version20240502101845.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MatchBot\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+use MatchBot\Application\Matching\Adapter;
+use MatchBot\Application\RealTimeMatchingStorage;
+use MatchBot\Domain\CampaignRepository;
+use Psr\Container\ContainerInterface;
+
+final class Version20240502101845 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Match fund corrections for MAT-361 campaigns';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql(<<<SQL
+            -- reduce amount from 5_000 to 2_500, reduce amountAvailable from 1_890 to 0
+            UPDATE CampaignFunding SET amount = amount - 2500, amountAvailable = amountAvailable - 1890 WHERE id = 29101 LIMIT 1;
+
+            -- reduce amount from 2_500 to 1890;
+            UPDATE FundingWithdrawal set amount = amount - 610 where id = 717813;
+            
+            -- reduce amount from 20_000 to 10_000, reduce amountAvailable from 9952 to 0
+            UPDATE CampaignFunding SET amount = amount - 10000, amountAvailable = amountAvailable - 9952 WHERE id = 29116 LIMIT 1;
+
+            -- reduce amount from 10_000 to zero
+            UPDATE FundingWithdrawal set amount = amount - 9952 where id = 709467 LIMIT 1;
```

Comment:
hmm now I'm thinking I went wrong somewhere because the 9952 is repeated in the previous line but the 610 further up wasn't repeated.

Reply:
I think 41feab0 fixes it.